.changeset/tender-donkeys-carry.md

```diff
@@ -0,0 +1,37 @@
+---
+"@salt-ds/core": minor
+---
+
+Added `Slider` and `RangeSlider` components.
```

Comment:
would delete the other changeset and change this to 

`Added `Slider` and `RangeSlider` components to  core.`

will read better